srml/im-online/src/lib.rs

```diff
@@ -0,0 +1,442 @@
+// Copyright 2017-2019 Parity Technologies (UK) Ltd.
```

Comment:
Do we have tests for this module? Or is it coming in a follow up PR?

Reply:
Will be a follow-up PR, I'm already working on it.